tests/test_dagmc_stats.py

```diff
@@ -71,20 +77,18 @@ def test_get_triangles_per_surface(self):
         """
         Tests some parts of the get_triangles_per_surface function
         """
-        dagmc_tags = ds.get_dagmc_tags(my_core)
+        dagmc_tags = ds.get_dagmc_tags(my_core_2)
         entityset_ranges = ds.get_entityset_ranges(
-            my_core, root_set, dagmc_tags['geom_dim'])
+            my_core_2, root_set_2, dagmc_tags['geom_dim'])
         t_p_s_data = ds.get_triangles_per_surface(
-            my_core, entityset_ranges)
-        surfaces = my_core.get_entities_by_type_and_tag(
-            root_set, types.MBENTITYSET, dagmc_tags['geom_dim'], [2]).size()
-        assert(len(t_p_s_data) == surfaces)
-
-        # !! This does not pass the tests and I do not know what it is supposed to do
-        # needs updating !
-
-        #triangles = my_core.get_entities_by_type(root_set, types.MBTRI).size()
-        #assert(sum(t_p_s_data) == triangles)
+            my_core_2, entityset_ranges)
+            
+        assert(len(t_p_s_data) == 6)
+        
+        assert(sum(t_p_s_data.values()) == 12)
+        
+        for v in t_p_s_data.values():
```

Comment:
numpy has a way to compare two arrays: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html